src/runtime/security/negotiation.cpp

```diff
@@ -32,6 +32,10 @@ const std::set<std::string> supported_mechanisms{"GSSAPI"};
 
 DSN_DEFINE_bool("security", enable_auth, false, "whether open auth or not");
 DSN_DEFINE_bool("security", mandatory_auth, false, "wheter to do authertication mandatorily");
+DSN_DEFINE_bool("security",
```

Comment:
Better to keep these configs in lexicographical order, it would be friendly for reading.